cloud/jenkins/ps_operator_aws_openshift-4.groovy

```diff
@@ -38,52 +38,25 @@ void runTest(String TEST_NAME) {
             echo "The $TEST_NAME test was started!"
             testsReportMap[TEST_NAME] = 'failure'
 
-            def FILE_NAME = "${env.GIT_BRANCH}-${env.GIT_SHORT_COMMIT}-$TEST_NAME-eks-${env.PLATFORM_VER}"
+            def FILE_NAME = "$GIT_BRANCH-$GIT_SHORT_COMMIT-$TEST_NAME-eks-$PLATFORM_VER-$PARAMS_HASH"
             popArtifactFile("$FILE_NAME")
 
             timeout(time: 90, unit: 'MINUTES') {
                 sh """
                     if [ -f "$FILE_NAME" ]; then
                         echo "Skipping $TEST_NAME test because it passed in previous run."
                     else
-                        cd ./source
-                        if [ -n "${OPERATOR_IMAGE}" ]; then
-                            export IMAGE=${OPERATOR_IMAGE}
-                        else
-                            export IMAGE=perconalab/percona-server-mysql-operator:${env.GIT_BRANCH}
-                        fi
-
-                        if [ -n "${IMAGE_MYSQL}" ]; then
-                            export IMAGE_MYSQL=${IMAGE_MYSQL}
-                        fi
-
-                        if [ -n "${IMAGE_ORCHESTRATOR}" ]; then
-                            export IMAGE_ORCHESTRATOR=${IMAGE_ORCHESTRATOR}
-                        fi
-
-                        if [ -n "${IMAGE_ROUTER}" ]; then
-                            export IMAGE_ROUTER=${IMAGE_ROUTER}
-                        fi
-
-                        if [ -n "${IMAGE_BACKUP}" ]; then
-                            export IMAGE_BACKUP=${IMAGE_BACKUP}
-                        fi
-
-                        if [ -n "${IMAGE_TOOLKIT}" ]; then
-                            export IMAGE_TOOLKIT=${IMAGE_TOOLKIT}
-                        fi
-
-                        if [ -n "${IMAGE_PMM}" ]; then
-                            export IMAGE_PMM=${IMAGE_PMM}
-                        fi
-
-                        if [ -n "${IMAGE_PMM_SERVER_REPO}" ]; then
-                            export IMAGE_PMM_SERVER_REPO=${IMAGE_PMM_SERVER_REPO}
-                        fi
-
-                        if [ -n "${IMAGE_PMM_SERVER_TAG}" ]; then
-                            export IMAGE_PMM_SERVER_TAG=${IMAGE_PMM_SERVER_TAG}
-                        fi
+                        cd source
+
+                        [[ "$OPERATOR_IMAGE" ]] && export IMAGE=$OPERATOR_IMAGE || export IMAGE=perconalab/percona-server-mysql-operator:$GIT_BRANCH
```

Comment:
What's the logic behind this? If we don't export operator image export everything else? Please make it so that we have it consistent because this is not an optimisation currently as it is and it is inconsistent also.

Reply:
All I have done is change from:
```
if [ -n "${OPERATOR_IMAGE}" ]; then
    export IMAGE=${OPERATOR_IMAGE}
else
    export IMAGE=perconalab/percona-server-mysql-operator:${env.GIT_BRANCH}
fi
```
to:
`[[ "$OPERATOR_IMAGE" ]] && export IMAGE=$OPERATOR_IMAGE || export IMAGE=perconalab/percona-server-mysql-operator:$GIT_BRANCH `
this is pure syntax sugar, no logic is changed here.